feature-libs/product-configurator/rulebased/occ/variant/converters/occ-configurator-variant-normalizer.ts

```diff
@@ -44,7 +44,7 @@ export class OccConfiguratorVariantNormalizer
       pricingEnabled: source.pricingEnabled ?? true,
       hideBasePriceAndSelectedOptions: source.hideBasePriceAndSelectedOptions,
       immediateConflictResolution: source.immediateConflictResolution ?? false,
-      newConfiguration: source.newConfiguration ?? true,
+      newConfiguration: source.newConfiguration, // we need a trinary state true, false, undefined!
```

Comment:
Why not set `newConfiguration` to `false` if `undefined`?
`newConfiguration: source.newConfiguration ?? false`

Reply:
then you would get the restart-dialog